deps/v8/src/log.cc

```diff
@@ -891,11 +892,13 @@ void Logger::ApiSecurityCheck() {
 
 void Logger::SharedLibraryEvent(const std::string& library_path,
                                 uintptr_t start,
-                                uintptr_t end) {
+                                uintptr_t end,
+                                uintptr_t slide) {
   if (!log_->IsEnabled() || !FLAG_prof_cpp) return;
   Log::MessageBuilder msg(log_);
-  msg.Append("shared-library,\"%s\",0x%08" V8PRIxPTR ",0x%08" V8PRIxPTR,
-             library_path.c_str(), start, end);
+  msg.Append("shared-library,\"%s\",0x%08" V8PRIxPTR ",0x%08" V8PRIxPTR
+                 ",0x%08" V8PRIxPTR,
```

Comment:
Any pros of using unsigned int and base 10 here? I like how things may work both ways here.